src/main/resources/Message_it.properties

```diff
@@ -0,0 +1,27 @@
+#
+# Strings used by the Messages.java enum (i.e., if that enum's name changes, the properties files' names must change)
+#
+
+# PLANNER ERROR MESSAGES
+PLAN_OK      = Successo
+SYSTEM_ERROR = Ci spiace. Il servizio è temporaneamente indisponibile. Riprovate più tardi.
+GRAPH_UNAVAILABLE = Ci spiace. Il servizio è temporaneamente indisponibile. Riprovate più tardi.

```

Comment:
Il grafo è temporaneamente indisponibile?

Reply:
The result of the graph being unavailable is that the service of OTP is not available so, from the end user perspective the service is down.
If we prefer to be more specific/technical we can say that the graph is unavailable, but we give for granted that the user understand the meaning. Also in the English version the text for the generic error and the one of the missing graph are the same probably because to the final user there is really no interested in the tech detail.
If we want to convey the different error reason we need to change on all languages and find a way to express the fact that there is a "graph related error"

Reply:
OK